shell.nix

```diff
@@ -17,5 +17,6 @@ pkgs.mkShell {
     pkgs.nixpkgs-fmt
     pkgs.poetry
     pkgs.niv
+    pkgs.jq
```

Comment:
Short question, where is `jq` used here?

Reply:
sorry, I just noticed this, it is used by the update script `pkgs/poetry/update`